### PR TITLE
Redesign all pages

### PR DESCRIPTION
**Pull request by iampinzao**

### CODE REVIEW

**Comment by jfberry** [comment]
Wow, there are 1800 files with this - is there a way to pull in these requirements somehow? Are they all needed?

**Comment by bbdoc** [comment]
> Wow, there are 1800 files with this - is there a way to pull in these requirements somehow? Are they all needed?

That was one of my point as well. Do we really need all this to be local. We could also directly reference bootstrap. Only advantage of having full bootstrap local is to allow running locally without any internet connection which I don't think anyone will ever do.

**Comment by iampinzao** [comment]
> Wow, there are 1800 files with this - is there a way to pull in these requirements somehow? Are they all needed?

And if somehow bootstrap goes down, as we have the framework locally, everything will keep running on our side .. but yeah, we could reference it directly instead of referencing the local files

**Comment by jfberry** [comment]
I didn’t say it was better just wondered if it could be built locally and ignored as part of the repository or simplified somehow.

—
James Berry, m. 07768 021031


________________________________

**Comment by jfberry** [comment]
Didn’t say one way was better than the other just wondered if it could be built locally (and part of git ignore) rather than included in the project

**Comment by bbdoc** [comment]
> Didn’t say one way was better than the other just wondered if it could be built locally (and part of git ignore) rather than included in the project

It is possible to build it locally using composer.

I discussed it with pin and I think the best option is to add a config option so people can chose between local or remote and default to remote. So people who want it local can build it and switch it to local in config.

File: session.php [hunk 1]
```diff
@@ -9,11 +9,21 @@
...
+
+   $conn = new mysqli($dbhost.":".$dbport, $dbuser, $dbpass, $db);
+   
+   // Check connection
+   if ($conn->connect_errno) {
+      echo "Failed to connect to MySQL: " . $conn->connect_error;
+      exit();
+   }
+
+   $sql = "SELECT * from humans WHERE id = '".$_SESSION['id']."'";
```
Comment (jfberry) [inline session.php:21]: Think this should be done with a prepared statement as injection attack possible here? Not sure if this is pervasive

File: package.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "PoraclWeb",
```
Comment (jfberry) [inline package.json]: PoracleWeb?

File: session.php (hunk 1 above)
Comment (bbdoc) [reply session.php:21]: Would be pretty difficult for an attacker to mess with a session variable in order to inject, but on the principle you are right... I need to change a few SQL to use prepare statements. Will do that but this is not really part of this PR

File: package.json (hunk 2 above)
Comment (bbdoc) [reply package.json]: corrected

File: session.php (hunk 1 above)
Comment (jfberry) [reply session.php]: Depends on the source of it and how it can be manipulated. Trust no-one :-)

**Pull request merged**